### ISSUE TITLE
Error when run in docker

### ISSUE THREAD

**Issue by nosovk** [Wladastic/Auto-GPT-Telegram-Plugin#18]
```
  File "plugins/Auto-GPT-Telegram-Plugin.zip/src/telegram_plugin/__init__.py", line 10, in <module>
  File "plugins/Auto-GPT-Telegram-Plugin.zip/src/telegram_plugin/telegram_chat.py", line 4, in <module>
ModuleNotFoundError: No module named 'telegram'
```
docker was run via:
```bash
docker compose run --rm auto-gpt --install-plugin-deps
```
it seems that `--install-plugin-deps` not compatible with plugin

<img width="608" alt="image" src="https://github.com/Wladastic/Auto-GPT-Telegram-Plugin/assets/1398808/e3fc2579-d573-48e0-814f-1c1481ca96d6">

**Comment by Wladastic** [comment]
Thats an issue with autogpt itself, there are a few prs open that will fix it

**Comment by nosovk** [comment]
if you mean that PR: https://github.com/Significant-Gravitas/Auto-GPT/pull/4151
then it will not help, because I was calling `--install-plugin-deps` already

**Comment by Wladastic** [comment]
Then go to your folder inside docker and install with pip install python-telegram-bot

**Comment by Wladastic** [comment]
I just got to check, apparently sometimes the plugin requirements cannot be installed on windows, even with docker.
I just got to reproduce your error and I think it was also fixed just now on auto-gpt, just pull the latest version for both.
If it doesn't help, login to you docker, go to app or the autogpt folder you configured and  run:
pip uninstall python-telegram-bot telegram
pip install python-telegram-bot

For some reason it installs "telegram" which is the wrong package, as it doesn't contain telegram.ext packages.
